tests/sentry/monitors/test_tasks.py

```diff
@@ -139,6 +139,13 @@ def test_missing_checkin_with_margin(self):
         ).replace(second=0, microsecond=0)
         assert missed_check.monitor_config == monitor.config
 
+        # Monitor environment next_checkin values are updated correctly
+        monitor_environment_updated = MonitorEnvironment.objects.get(id=monitor_environment.id)
+        assert (
+            monitor_environment_updated.next_checkin + timedelta(minutes=5)
+            == monitor_environment_updated.next_checkin_latest
```

Comment:
typically I would write the assert the other way around
```suggestion
            monitor_environment_updated.next_checkin_latest
            == monitor_environment_updated.next_checkin + timedelta(minutes=5)
```